plenum/server/consensus/msg_validator.py

```diff
@@ -20,6 +19,8 @@ def has_already_ordered(self, view_no, pp_seq_no):
                                 self._data.last_ordered_3pc) >= 0
 
     def validate(self, msg):
+        if isinstance(msg, NewViewCheckpointsApplied):
+            return PROCESS, None
```

Comment:
This looks kind of hacky, but probably we can live with it for a while

Reply:
Sure, this class will be removed pretty soon.